discovery-server.repos

```diff
@@ -3,9 +3,9 @@ repositories:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
         version: master
-    eProsima/Fast-RTPS:
+    eProsima/Fast-DDS:
```

Comment:
I suggest unifying and using the same default folder name as in Fast DDS:

```suggestion
    eProsima/fastdds:
```

If accepted, I suggest applying the same criteria to the other packages too

Reply:
If we are applying these changes, I would suggest to also remove the folder eProsima/

Reply:
After internal meeting we decide to apply this suggestion